### PR TITLE
Issue #40: Added Travel Page

### PR DESCRIPTION
**Pull request by aditya-prayaga**
@ananyo2012  Please Verify the content.

### CODE REVIEW

**Comment by ananyo2012** [comment]
@aditya-369 You can write fully in markdown instead of using html tags

File: travel.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+<p>Alternatively, you can hire metered air-conditioned radio cabs. Skycabs and Meru are approved by the airport itself @ 21 INR/km and are available just after exiting the terminal building. For the rest, you need to call and book with a lead time of 15 minutes to 1 hour. These cabs charge 25% surcharge in the night (ie, between 23:00 hours and 05:00 hours). Hyderabad traffic police counter is on the ground floor with prepaid taxis.</p>
+
+<p><b>Beware of taxi soliciting tours at the airport greeting area; they might try to scam you into paying exorbitant rates (especially so in case of non-locals)</b>. The Hyderabad Traffic Police has partnered with the Airport authorities to run a counter for prepaid taxis. This is a safe option as all taxis are registered with the police. The police also runs a <b>SHE cab service</b> now, especially for women travellers, with female drivers and safety equipment such as a GPS connected to the Police Control Room.</p>
+
+<p>One can also hire a cab from many app-based services such as Uber or Ola directly to the Venue which are readily available in Hyderabad.</p>
```
Comment (ananyo2012) [inline travel.md]: Bold Uber, Ola

File: travel.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+* [Trains to Secundrabad](https://trainspy.com/static/station/(SC)SECUNDERABAD-JN)
+* [Trains to Hyderabad - Deccan](http://trainspy.com/static/station/(HYB)HYDERABAD-DECAN)
+* [Trains to Kachiguda](http://trainspy.com/static/station/(KCG)KACHEGUDA)
+
+<p>Secundrabad and Kachiguda are almost the same distance from the venue (~40 kms). Hyderabad (Nampally) Station is ~15 kms from the venue. Using the above links, you can check which trains you can get from your region.</p>
+
+<p><b>From the stations to the venue:</b> From any station you have can take a bus to Gachibowli (ORR). Just point Google Map from your location to Gachibowli and you will get the bus numbers you need to board from the station bus stop. From Gachibowli you can take a cab or auto reach the venue.</p>
+
+<p>You can also get a cab from these stations to the venue directly. Ola and Uber are 2 popular cab services in the area. You can use their app to book a cab.</p>
```
Comment (ananyo2012) [inline travel.md]: Bold Uber, Ola

File: travel.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+<p>Indian Railways has service to Hyderabad from all over India. There are three major railway stations serving the twin cities: <b>Secunderabad, Hyderabad , Kachiguda</b> and a minor station at <b>Begumpet</b>. Hyderabad Deccan Station is popularly known as <b>Nampally Station</b>. Most of the trains bound for South India and North India originate from Hyderabad,and leave via Secunderabad.</p>
+
+* [Trains to Secundrabad](https://trainspy.com/static/station/(SC)SECUNDERABAD-JN)
+* [Trains to Hyderabad - Deccan](http://trainspy.com/static/station/(HYB)HYDERABAD-DECAN)
+* [Trains to Kachiguda](http://trainspy.com/static/station/(KCG)KACHEGUDA)
+
+<p>Secundrabad and Kachiguda are almost the same distance from the venue (~40 kms). Hyderabad (Nampally) Station is ~15 kms from the venue. Using the above links, you can check which trains you can get from your region.</p>
+
+<p><b>From the stations to the venue:</b> From any station you have can take a bus to Gachibowli (ORR). Just point Google Map from your location to Gachibowli and you will get the bus numbers you need to board from the station bus stop. From Gachibowli you can take a cab or auto reach the venue.</p>
```
Comment (ananyo2012) [inline travel.md]: Can you add a section for metro as well. From Secunderabad they need to take the Metro blue line to Hitech City. From Nampally they need to take Red line and change to blue line at Ameerpet and get down at Hi-Tech city metro stop. Here is the link to metro map https://www.ltmetro.com/metro-network-map/

File: travel.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+<p><b>MGBS or Imliban,</b>is said to be the largest bus station in the world with around 84 bus bays side by side. TSRTC has pickup and drop points from various points in the city. In addition, government-run bus services of neighboring states also run buses to Hyderabad, as do various private companies.</p>
+
+<p><b>Private Buses</b>South India is largely well served by organised private bus operators. They run luxury buses like Volvo, Mercedes, Kinglong Cerita buses including multi axle buses. These are air-conditioned, semi sleeper or sleeper services with online ticket booking facilities. Important private travel hubs are KPHB Colony, Lakdi-Ka-Pool, Paradise centre in Secunderabad and Dilsukh nagar. Luxury services run to many cities from these places. Non-metropolitan areas and towns are often connected by non-A/C buses but are still provided with comfortable seating.</p>
```
Comment (ananyo2012) [inline travel.md]: Put a space here after the heading

File: travel.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+## By Bus 
+<p><b>To Hyderabad:</b> Hyderabad is well-connected to all parts of the Telangana and most parts of South and Western India. Both state government and private buses operate large number of luxury and ordinary services across the state and neighboring states.</p>
+
+<p><b>JBS, (Jubilee Bus station),</b> is in Secunderabad. TSRTC runs direct A/C coaches to Mumbai, Bengaluru (Bangalore) and Chennai. Telangana tourism runs A/C coaches to, Mumbai, Chennai and Bengaluru. You have to book the tickets in advance. There are many online bus ticketing portals to book bus tickets. Mybustickets is one such portal.</p>
+
+<p><b>MGBS or Imliban,</b>is said to be the largest bus station in the world with around 84 bus bays side by side. TSRTC has pickup and drop points from various points in the city. In addition, government-run bus services of neighboring states also run buses to Hyderabad, as do various private companies.</p>
```
Comment (ananyo2012) [inline travel.md]: Put a space after the heading

File: travel.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+<p>It may be difficult to find direct buses from North India due large size of that part of the country.</p>
+
+<b>From the Bus stand to the venue:</b>
+* [JBS to Gachibowli City (IIIT Hyderabad)](https://www.google.co.in/maps/dir/Jubilee+Bus+Station,+Gandhi+Nagar,+Nehru+Nagar+Colony,+West+Marredpally,+Secunderabad,+Telangana+500026/Bus+stop+IIITH,+Old+Mumbai+Highway,+TCS+Synergy+park,+Indian+Immunologicals+Colony,+Gachibowli,+Hyderabad,+Telangana/@17.4151085,78.3937167,13z/data=!3m1!4b1!4m14!4m13!1m5!1m1!1s0x3bcb9a3e20505cdb:0xa6d0eb22d13169c6!2m2!1d78.4981525!2d17.4477577!1m5!1m1!1s0x3bcb935d9dcbf001:0x88b77ba3e0599247!2m2!1d78.3511163!2d17.4464139!3e3)
+* [MGBS to Hi-Tech City (IIIT Hyderabad)](https://www.google.co.in/maps/dir/MGBS+Bus+Stand,+Gowliguda+North,+Gowliguda,+Hyderabad,+Telangana/Bus+stop+IIITH,+Old+Mumbai+Highway,+TCS+Synergy+park,+Indian+Immunologicals+Colony,+Gachibowli,+Hyderabad,+Telangana/@17.4122356,78.3829749,13z/data=!3m1!4b1!4m14!4m13!1m5!1m1!1s0x3bcb982d09e67201:0x22cc2ebd6842c8be!2m2!1d78.4842753!2d17.3787197!1m5!1m1!1s0x3bcb935d9dcbf001:0x88b77ba3e0599247!2m2!1d78.3511163!2d17.4464139!3e3)
```
Comment (ananyo2012) [inline travel.md]: This should be MGBS to Gachibowli City as well

File: travel.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+<p>From the drop point you could walk to the venue.</p>
+<p>You can also get a cab from these bus stops to the venue directly.</p>
+<p><b>If you want more details or have any inquiries, please contact <u>contact@in.pycon.org.</u></b></p>
```
Comment (ananyo2012) [inline travel.md]: Update the email here - contact@hydpy.org and put a mailto link here

**Review by ananyo2012 — COMMENTED** [review]
Please update the content as per comments

**Comment by bhansa** [comment]
@aditya-369 please update the comments.

**Comment by aditya-prayaga** [comment]
Sure I will raise a PR within two days

**Comment by aditya-prayaga** [comment]
@ananyo2012 Please Review PR made some Updates.

**Review by ananyo2012 — APPROVED** [review]
LGTM. Will merge in sometime

**Pull request merged**